src/content/docs/en/pages/devtools/cli/azion-cli-alpha/init-build-deploy/azion-create-deploy-application.mdx

```diff
@@ -1,12 +1,16 @@
 ---
-title: Azion CLI's init, build, and deploy commands
+title: Azion CLI init, build, and deploy commands
 description: Create and deploy a new application from Azion's templates.
 meta_tags: cli, next, javascript, astro, vue, react, edge computing
 namespace: documentation_cli_edge_applications_journey
 menu_namespace: cliMenuAlpha
 permalink: /documentation/products/azion-cli/create-application/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```